_includes/team-section.html

```diff
@@ -0,0 +1,148 @@
+<div class="team-section-container container">
+  <h2 class="team-title">TEAM</h2>
+  <div>
+    <div class="col-lg-4 col-md-6">
+      <div class="card">
+        <img class="card-image" src="{{site.team_headshots_uri | relative_uri}}Eric_Hambro.png" />
+        <div class="card-body">
+          <h5 class="card-title">Eric Hambro</h5>
+          <p class="card-text">Research Engineer (FAIR)</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-4 col-md-6">
+      <div class="card">
+        <img class="card-image" src="{{site.team_headshots_uri | relative_uri}}Sharada_Mohanty.png" />
+        <div class="card-body">
+          <h5 class="card-title">Sharada Mohanty</h5>
+          <p class="card-text">CEO & Co-Founder (AIcrowd)</p>
+        </div>
+      </div>
```

Comment:
Is it possible to abstract this block out so that the variables can be passed as attributes? That way, if we need to change the styles on the team member card in the future, we do not have to update 20 different blocks, all individually. @Bradmin82 : Any thoughts?

Reply:
### I think this is a nice idea, but. . .
While it is possible to [create components in jekyll](https://github.com/helpscout/jekyll-components/blob/master/docs/creating-a-component.md), given our time frame and unfamiliarity w/ jekyll components I believe this may be too risky for launch. This is something that should be considered post-launch, especially if the team expects to grow. Currently, there are only 15 team members, which isn't an overwhelming amount of cards to update.

### Updating all the cards at once
Every element of the team-card has a unique class that can be targeted to style all of the blocks at once. That means that each card's padding or card background-color could be updated in one place with one line of css. We have separated the styles for the team section into its own stylesheet so that they are easy to find and update, all in one place: /theme-css/team-card.css